_source/logzio_collections/_log-sources/lambda-extensions.md

```diff
@@ -0,0 +1,213 @@
+
+Add the environment variables to the function, according to the environment variables table in the last tab of this topic.
+
+##### Run the function
+
+Run the function. It may take more than one run of the function for the logs to start shipping to your Logz.io account.
+
+</div>
+
+#### Deleting the extension
+
+- To delete the **extension layer**: In your function page, go to the **layers** panel. Click `edit`, select the extension layer, and click `save`.
+- To delete the **extension dependencies layer**: In your function page, go to the **layers** panel, click `edit`, select the extension dependencies layer, and click `save`.
+- To delete the extension's **environment variables**: In your function page, select the `Configuration` tab, select `Environment variables`,  click `edit`, and remove the variables that you added for the extension.
+
+
+</div>
+<!-- tab:end -->
+
+<!-- tab4:start -->
+<div id="tables">
+
+### Environment Variables
+
+| Name | Description |Required/Default|
+| --- | --- | --- |
+| `LOGZIO_LOGS_TOKEN` | Your Logz.io logs shipping [token](https://app.logz.io/#/dashboard/settings/general). | Required |
+| `LOGZIO_REGION` |  Two-letter region code, or blank for US. This determines your listener URL. You can find your region code in the [Regions and URLs table](https://docs.logz.io/user-guide/accounts/account-region.html#regions-and-urls). | Default: (left blank for US)|
+| `LOGS_EXT_LOG_LEVEL` |  Log level of the extension. Can be set to one of the following: `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`. |Default: `INFO` |
+| `ENABLE_EXTENSION_LOGS` |  Set to `true` if you wish the extension logs will be shipped to your Logz.io account. | Default: `false` |
+| `ENABLE_PLATFORM_LOGS` | The platform log captures runtime or execution environment errors. Set to `true` if you wish the platform logs will be shipped to your Logz.io account. | Default: `false` |
+| `THREAD_TIMEOUT` | Execution timeout for the threads that parse and send the logs. | Default: `5` (seconds)|
+| `LOGZIO_CUSTOM_LISTENER` | Use if you have a custom Logz.io listener endpoint). Will override `LOGZIO_REGION`. | Optional |
+
+
+### Lambda extension versions
+
+| Version | Supported Runtimes | AWS ARN |
+| --- | --- | --- |
+| 0.0.1 | python 3.7, python 3.8 | `arn:aws:lambda:<<YOUR-AWS-REGION-CODE>>:486140753397:layer:LogzioLambdaExtensionLogs:1` |
+
+### ARN for extension dependencies
+
+|Compatible with extension versions | Imports | AWS ARN |
+| --- | --- | --- |
+| 0.0.1 | `requests` | `arn:aws:lambda:<<YOUR-AWS-REGION-CODE>>:486140753397:layer:LogzioLambdaExtensionLogsLibs:1` |
+
+### Available AWS regions
+
+| Region Name | Region Code |
+| --- | ---- |
+| US East (N. Virginia) | `us-east-1` |
+| US East (Ohio) | `us-east-2` |
+| US West (N. California) | `us-west-1` |
+| US West (Oregon) | `us-west-2` |
+| Europe (Frankfurt) | `eu-central-1` |
+| Europe (Ireland) | `eu-west-1` |
+| Europe (Stockholm) | `eu-north-1` |
```

Comment:
@Doron-Bargo Are there more regions you'd want me to add for the initial release?

Reply:
@mirii1994 - i think we can start with that and add note that they should contact support if they wants more regions

Reply:
@Doron-Bargo great, already added that note, thanks :)